Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html

```diff
@@ -11,7 +11,7 @@
   </div>
   <div *ngIf="!imageLoaded" class="image-placeholder" #placeholder>
     <svg height="30" width="100" viewBox="0 0 100 30">
-      <text x="50" y="15" fill="black" font-size="30" text-anchor="middle" dominant-baseline="central">
+      <text x="50" y="25" fill="black" font-size="30" text-anchor="middle">
```

Comment:
maybe okay... Can someone override the SVG height and width? or the text size?

Reply:
of course, you could, but what for?

Reply:
can SVG use variabels? Then the y and the x of the text should be depend on the size of the svg and on the text size.

Reply:
lets discuss this in person.

Reply:
... and there was a reasone for this beeing SVG, scalability :)